rust/private/rustc.bzl

```diff
@@ -671,7 +671,12 @@ def collect_inputs(
     """
     linker_script = getattr(file, "linker_script") if hasattr(file, "linker_script") else None
 
-    linker_depset = cc_toolchain.linker_files()
+    # TODO(gnish): rules_rust is not coupled with Bazel release. Remove conditional and change to
```

Comment:
Can you instead reference a public github issue or something? The Google `TODO(name)` comments are not that useful for external folks.

Reply:
Unfortunately there is no public github issue - I will remove TODO. I think the comment is useful even without an issue it clearly explains why this change is necessary and when it should be removed. I can extend it if you'd like to.

Reply:
I removed username, and updated the comment to better describe the issue.

Reply:
Done.